katalog/karma/README.md

```diff
@@ -0,0 +1,47 @@
+# Karma
+
+<!-- <KFD-DOCS> -->
+
+Karma is a UI for Alertmanager, useful for browsing alerts based on labels and managing silences.
+It can also aggregate alerts from multiple Alertmanager instances.
+
+*Source:* [prymitive/karma][k-gh]
+
+## Requirements
+
+- Kubernetes >= `1.21.0`
+- Kustomize `= v3.5.3`
+- [prometheus-operator](../prometheus-operator)
+- [prometheus-operated](../prometheus-operated)
+- [alertmanager-operated](../alertmanager-operated)
+
+## Image repository and tag
+
+- Karma image: `registry.sighup.io/fury/karma/karma:v0.113`
+- Karma repository: [Karma on GitHub][k-gh]
+
+## Configuration
+
+Fury distribution Karma is deployed with the following
+configuration:
+- Alertmanager URI: `http://alertmanager-main.monitoring.svc.cluster.local:9093`
+- Polling interval: `1m`
+
+All configuration options can be found [here](https://github.com/prymitive/karma/blob/main/docs/CONFIGURATION.md)
+
+## Deployment
+You can deploy karma by running the following command:
```

Comment:
linting
```suggestion
## Deployment

You can deploy karma by running the following command:
```

maybe we can say something about how to create the configuration patch here?